src/otx/core/model/base.py

```diff
@@ -100,10 +99,12 @@ class OTXModel(LightningModule, Generic[T_OTXBatchDataEntity, T_OTXBatchPredEnti
     """
 
     _OPTIMIZED_MODEL_BASE_NAME: str = "optimized_model"
+    input_size_multiplier = 1
```

Comment:
In my opinion, it would be nice to have a docstring for this, this is a variable that doesn't make sense at first glance.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3788/commits/0adb7ea54bb57b761f75b423167b45772d08d416